project/src/test/java/com/google/sps/data/DataAccessorTest.java

```diff
@@ -162,6 +261,34 @@ public void storeUserPreferences_emptydUserId_throwIllegalArgumentException() {
         () -> dataAccessor.storeUserPreferences("", userPrefs));
   }
 
+  private Entity createRecomEntityByProperties(String userId, String placeId, boolean chosen,
+  boolean tryAgain) {
+    Entity recommendationEntity = new Entity(DataAccessor.RECOMMENDATION_ENTITY_KIND);
+    recommendationEntity.setProperty(DataAccessor.USER_ID_PROPERTY, userId);
+    recommendationEntity.setProperty(DataAccessor.PLACE_ID_PROPERTY, placeId);
+    recommendationEntity.setProperty(DataAccessor.TIME_PROPERTY, System.currentTimeMillis());
+    recommendationEntity.setProperty(DataAccessor.TRY_AGAIN_PROPERTY, tryAgain);
+    recommendationEntity.setProperty(DataAccessor.CHOSEN_PROPERTY, chosen);
+    return recommendationEntity;
+  }
+
+  private List<Entity> getRecommendationEntitiesByUserId(String userId) {
```

Comment:
replace `get` with `fetch` to make it clearer that this is calling the data store.

Reply:
Great idea, it's much clearer now. done